WordPress/src/main/java/org/wordpress/android/util/EncryptionUtils.java

```diff
@@ -0,0 +1,121 @@
+package org.wordpress.android.util;
+
+import android.content.Context;
+import android.os.Environment;
+import android.util.Base64;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.libsodium.jni.NaCl;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class EncryptionUtils {
+    static final int BOX_SEALBYTES = NaCl.sodium().crypto_box_sealbytes();
+    static final int XCHACHA20POLY1305_KEYBYTES = NaCl.sodium().crypto_secretstream_xchacha20poly1305_keybytes();
+    static final int XCHACHA20POLY1305_STATEBYTES = NaCl.sodium().crypto_secretstream_xchacha20poly1305_statebytes();
+    static final int XCHACHA20POLY1305_HEADERBYTES = NaCl.sodium().crypto_secretstream_xchacha20poly1305_headerbytes();
+    static final int XCHACHA20POLY1305_ABYTES = NaCl.sodium().crypto_secretstream_xchacha20poly1305_abytes();
+    static final int BASE64_ENCODE_FLAGS = Base64.DEFAULT;
+
+    static final String PUBLIC_KEY_BASE64 = "K0y2oQ++gEN00S4CbCH3IYoBIxVF6H86Wz4wi2t2C3M=";
```

Comment:
Can we avoid hard-coding the public key in the class? Different apps implementing this code may wish to use different keys, so it'd be nice to be able to configure this from the calling app without editing this file (assuming it remains in the app code).

Reply:
Got it!  I will make this change. Thank you! :)

Reply:
@jkmassel Would [`key_strings.xml`](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/res/values/key_strings.xml) be a good place for us to specify the public key?

Reply:
the public key will now be a parameter.  the change is here: 77d4e2eb68ad1c3e87a6c1531c27e41f5d70fe90